### PR TITLE
Add rhevm guest add/del/exist/start/stop/suspend functions

### PR DESCRIPTION
**Pull request by hkx303**
**Description**
Add guest add/del/exist/start/stop/suspend functions

### CODE REVIEW

File: hypervisor/virt/rhevm/rhevmcli.py [hunk 1]
```diff
@@ -182,3 +198,214 @@ def get_guest_mac(self, guest_name):
...
+        if ret == 0 and "name" in output:
+            nic = output.strip().split(': ')[1].strip()
+            logger.info(f"rhevm({self.server}) guest nic is: {nic}")
+            return nic
+        else:
+            raise FailException(f"Failed to check rhevm({self.server}) guest nic")
+
+    def guest_add(self, guest_name, template, cluster, disk, host_name=None):
+        """
+        Create a new virtual machine.
```
Comment (Junefen) [inline hypervisor/virt/rhevm/rhevmcli.py:296]: It's better to give a detailed introduction about the we should prepare a template if we want to use the function to add guest.

File: hypervisor/virt/rhevm/rhevmcli.py [hunk 2]
```diff
@@ -182,3 +198,214 @@ def get_guest_mac(self, guest_name):
...
+            logger.info(f"rhevm({self.server}) guest {guest_name} is not exist")
+            return False
+
+    def guest_start(self, guest_name, host_name=None):
+        """
+        Power on virtual machines.
+        :param guest_name: the virtual machines you want to power on.
+        :param host_name: the name of the rhevm host
+        :return: power on successfully, return True, else, return False.
+        :param guest_name:
```
Comment (Junefen) [inline hypervisor/virt/rhevm/rhevmcli.py]: useless line

File: hypervisor/virt/rhevm/rhevmcli.py (hunk 1 above)
Comment (hkx303) [reply hypervisor/virt/rhevm/rhevmcli.py:296]: Yes! Added.

File: hypervisor/virt/rhevm/rhevmcli.py (hunk 2 above)
Comment (hkx303) [reply hypervisor/virt/rhevm/rhevmcli.py]: Removed it

**Pull request merged**